webdriver/support/merge_dictionaries.py

```diff
@@ -0,0 +1,42 @@
+def merge_dictionaries(first, second):
```

Comment:
If a given capability is specified by both the environment and the test, it's
not obvious which value should take precedence. We could just choose one or the
other, but I'm concerned our choice might conflict with some people's
expectations, and the resulting behavior could be hard to understand. It seems
safer to disallow over-specification all together.